tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+import theme from 'tailwindcss/defaultTheme.js';
+
+const config = {
+  darkMode: 'class',
+  theme,
+};
```

Comment:
just for my understanding: where is `prefix` here?

I also get error on master right now
https://studio.backlight.dev/review/wzPn6NO54tDJlBxCtjGx/tailwind-showcases/doc/tailwind-showcases.mdx?p=doc

![image](https://user-images.githubusercontent.com/137844/198297967-4d2285bf-9c13-412b-8447-caa6689bf8e9.png)

Reply:
Previously it didn't support at all prefix.
But I have another issue now

Reply:
do you get same error on master?